.travis.yml

```diff
@@ -32,8 +32,6 @@ before_install:
   - eval $(opam env)
   - export OPAMVERBOSE=1
 install:
-  # ensure infer isn't installed in this switch, then deal with dependencies
-  - opam remove infer
```

Comment:
The `opam remove` could be moved to the `script` job instead, so that it runs right before trying to `opam install infer`? The problem is that the current `before_cache` part doesn't always run, since it only happens when the build is successful, but I found that opam might still want to be cleaned up by running `opam remove infer` after a failed install.

Reply:
@jvillard you seems to review the older version. I rebased and removed this change, please refresh the page or whatever required.

Reply:
Sorry folks, I think this was just a very old comment I made and somehow never sent until yesterday.